src/rgw/rgw_sal_motr.h

```diff
@@ -962,11 +1023,87 @@ class MotrStore : public Store {
     int do_idx_op_by_name(std::string idx_name, enum m0_idx_opcode opcode,
                           std::string key_str, bufferlist &bl, bool update=true);
     int check_n_create_global_indices();
+    int store_access_key(const DoutPrefixProvider *dpp, optional_yield y, MotrAccessKey access_key);
+    int delete_access_key(const DoutPrefixProvider *dpp, optional_yield y, std::string access_key);
+    int store_email_info(const DoutPrefixProvider *dpp, optional_yield y, MotrEmailInfo& email_info);
 
     int init_metadata_cache(const DoutPrefixProvider *dpp, CephContext *cct);
     MotrMetaCache* get_obj_meta_cache() {return obj_meta_cache;}
     MotrMetaCache* get_user_cache() {return user_cache;}
     MotrMetaCache* get_bucket_inst_cache() {return bucket_inst_cache;}
 };
 
+struct obj_time_weight {
```

Comment:
This is copied from [rgw_rados.cc](https://github.com/zalsader/ceph/blob/master/src/rgw/rgw_rados.cc#L3682), it might be better to put it in a common header

Reply:
I don't know where the best place is, but maybe rgw_sal.h, or its own header. It's a nitpick of course, but the daos code might also use that in the future.